### ISSUE TITLE
Gravity block attached to child entity doesn't effect other entities.

### ISSUE THREAD

**Issue by CurioInventorium** [tsunamayo/Starship-EVO#3056]
### Short Description of the issue:
While experimenting with the gravity block for a new tutorial video I found that a gravity block attached to a child object on a rotator doesn't seem to effect entities. I think the player entity will rotate to orientate itself, but it doesn't get pushed or moved.
...

### Steps to reproduce the issue:
Place down a rotator, build some blocks onto the rotator and attach a gravity block. Configure gravity block in a direction other than the environment gravity. The player or other entities don't respond.
...

### Environment
- Starship EVO version: 20w41c
- Graphics Card: Nvidia Geforce RTX2060

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_
I've uploaded an unlisted video log to youTube: https://youtu.be/Xgr7i6phL-c
Attached picture to show layout used:
![20201010134446_1](https://user-images.githubusercontent.com/51359034/95656535-ef740700-0b06-11eb-8e97-d8a70980e4f0.jpg)

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5359093/output_log.txt)

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`

**Comment by CurioInventorium** [comment]
There also seems to be the situation were an object needs to be close to the gravity block to be effected by it when it turns on. Even if it's within the range of influence to begin with.
Say at 15 distance from gravity block, nothing happens when it's switched on. Only when about 2 or 3 distance from gravity block will an object respond when the gravity it turned on.

**Comment by tsunamayo** [comment]
@CurioInventorium yes it is because I am only checking the center of the entity, not every point...
Otherwise that would too complicated and too expensive. It is more made so you can have your smaller ship land on a bigger one.
thx

**Comment by CurioInventorium** [comment]
Okay, I understand that. That's what I figured and I wasn't asking for more. It's just that even when the centre of entity is within the defined gravity area, the entity only responds when it is close to the gravity block itself.
And entities don't seem to respond if the gravity block is on a child object, as shown in the video.

It is a wonderful build system you've created and I'm having fun exploring all it's possibilities. Thank you

**Comment by Uncle-Ulty** [comment]
Hi,

I found the same problem. a Gravity block on a CE over a hinge doesn't effect any object inside its field. It seems the bug persists
